services/core-web/src/styles/components/PermitConditions.scss

```diff
@@ -111,6 +111,28 @@ div.condition-layer {
   align-items: center;
 }
 
+.condition-tag-select {
```

Comment:
I'm wondering if this styling is something we can move into the multiselect component?
Something like where we pass in a prop like `tagMode=true` and it handles putting the classname on it. Just so we can easily use it in other places and keep it consistent